clients/vault/src/oracle/storage/traits.rs

```diff
@@ -163,11 +163,8 @@ pub trait ArchiveStorage {
 
 pub(crate) async fn download_file_and_save(url: &str, file_name: &str) -> Result<(), Error> {
 	let response = reqwest::get(url).await.map_err(|e| Error::ArchiveError(e.to_string()))?;
-	if response.status().is_server_error() | response.status().is_client_error() {
-		return Err(Error::ArchiveResponseError(format!("{response:?}")));
-	}
-
```

Comment:
Do we really want to remove this?

Reply:
I don't remember a reason to remove or touch this part. Probably part of the all same merge issue.